politeiawww/api/cms/v1/v1.go

```diff
@@ -108,7 +110,7 @@ type InvoiceRecord struct {
 	Version            string         `json:"version"`                      // Record version
 	ContractorName     string         `json:"contractorname"`               // IRL name of contractor
 	ContractorLocation string         `json:"contractorlocation"`           // IRL location of contractor
-	ContractorEmail    string         `json:"contractoremail"`              // Contractor email contact
+	ContractorEmail    string         `json:"contractorcontact"`            // Contractor email contact
```

Comment:
The JSON name should match the Structure field. If I read the rest of the PR right you went with `ContractorContact` Please rename `ContractorEmail to `ContractorContact`